addons/forum-quote-code-beautifier/addon.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Customizable quotes and code blocks on forums",
+  "name": "Modernize forum quotes",
```

Comment:
I'm not sure users will understand what "modernize" means here.

Reply:
Any other ideas? The current name isn't great because they're not that customizable.

Reply:
> Any other ideas? The current name isn't great because they're not that customizable.

Could they be called 3.0 quotes and code blocks?

Reply:
I'm not sure if the word blocks should be used since there's also scratchblocks.
How about
```suggestion
  "name": "Better forum quote style",
```
And maybe you want the S here too?

Reply:
Yes, that with the "s" would be great.

Reply:
@Hans5958 What do you think about the forum vs forum**s** thing? See https://github.com/ScratchAddons/ScratchAddons/pull/6582#discussion_r1303359413.

Reply:
Not totally sure about "better forum quote style", and I'm also not sure what to call it other than "modernize<b>d</b> forum quotes".

Scratch's current quote block style is also pretty likable and seems like a style Scratch would still use so I don't how calling it "better" or "modernized" would go with users.

Reply:
```suggestion
  "name": "Alternative forum quote style",
```